mods/ra2/audio/voices.yaml

```diff
@@ -193,6 +193,14 @@ NighthawkVoice:
 		Attack: vblhata, vblhatb, vblhatc, vblhatd
 		Die: vblhdia, vblhdib, vblhdic
 
+OspreyVoice:
```

Comment:
I can't null

```salt
	Aircraft:
		Voice:
```

It will always fallback to https://github.com/OpenRA/ra2/blob/4438a984780d153017969a85c5717e52c6ec8003/mods/ra2/rules/defaults.yaml#L920-L921

Reply:
The workaround I have to use now is worse imho.

Reply:
Hm, yes that is probably worse. Does "-Voice:" not work? My initial idea was that we likely shouldn't use `Voiced` at all since that unit doesn't have a voice in a usual sense but rather something that would play as sound of a unit explosion?

Reply:
```salt
		-Voice:
		Voice:
```

will then make it default to https://docs.openra.net/en/latest/release/traits/#aircraft `Action`

Reply:
I think https://docs.openra.net/en/latest/release/traits/#deathsounds is already pretty much the right trait. Why reinvent the wheel when every other aircraft is using this setup? https://github.com/OpenRA/ra2/blob/master/mods/ra2/audio/voices.yaml#L262

Reply:
Removed it here and left a note at https://github.com/OpenRA/ra2/issues/716 as I don't have a solution yet.

Reply:
We did exactly the same thing for https://github.com/OpenRA/OpenRA/blob/0135dd9ed33794c9b22a7078ba54564a5cc8a59a/mods/ts/audio/voices.yaml#L189-L193